docs/scos/dev/back-end-development/extending-spryker/development-strategies/project-modules/add-a-new-module.md

```diff
@@ -170,7 +170,21 @@ class HelloWorldFacade extends AbstractFacade implements HelloWorldFacadeInterfa
 }
 ```
 
-4. Modify the controller so that it calls the method added to your facade:
+```php
+<?php
+
+namespace Pyz\Zed\HelloWorld\Business;
+
+interface HelloWorldFacadeInterface
+{
+    /**
+     * @return string
```

Comment:
Do be very precise (at least in the documentation we should be) we should add the @api tag and the Specification as well

Reply:
@fscholzspryker do we someone's assistance here to fix this? I'm no sure what to add and where.

Reply:
I added a suggestion - here you find also another example from core: https://github.com/spryker/event/blob/master/src/Spryker/Zed/Event/Business/EventFacadeInterface.php#L18